### PR TITLE
.format() converted to f-strings in io, morphology subpackage

### PR DESCRIPTION
**Pull request by harshit-dxt**
## Description
Partially addresses Issue #5474 

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: skimage/morphology/_skeletonize.py [hunk 1]
```diff
@@ -84,8 +84,8 @@ def skeletonize(image, *, method=None):
     elif image.ndim == 3 or (image.ndim == 2 and method == 'lee'):
         skeleton = skeletonize_3d(image)
     else:
-        raise ValueError('skeletonize requires a 2D or 3D image as input, '
-                         'got {}D.'.format(image.ndim))
+        raise ValueError(f'skeletonize requires a 2D or 3D image as input, '
+                         'got {image.ndim}D.')
```
Comment (grlee77) [inline skimage/morphology/_skeletonize.py:88]:
```suggestion
        raise ValueError(f'skeletonize requires a 2D or 3D image as input, '
                         f'got {image.ndim}D.')
```

An f is also needed here.

**Review by grlee77 — DISMISSED** [review]
Thanks @harshit-dxt. There is one small fix needed, but otherwise this looks good to me.

**Comment by harshit-dxt** [comment]
Thanks @grlee77 for the response. I just made the change. Hope that fixes it all.

**Review by soupault — APPROVED** [review]

**Review by hmaarrfk — APPROVED** [review]

**Comment by hmaarrfk** [comment]
Thanks!

**Pull request merged**